perf/lock.ts

```diff
@@ -0,0 +1,93 @@
+import {assert} from './replicache.js';
+import {newTab} from '../src/test-util';
+import {uuid} from '../src/sync/uuid.js';
+
+enum Mode {
+  SameTab = '',
+  OtherTab = 'other tab',
+  OtherProcess = 'other process',
+}
+
+export function benchmarks() {
+  return [
+    benchmarkUncontendedRead(),
+    benchmarkHeldRead(Mode.SameTab),
+    benchmarkHeldRead(Mode.OtherTab),
+    benchmarkHeldRead(Mode.OtherProcess),
+    benchmarkUncontendedWrite(),
+  ];
+}
+
+export async function isHeld(name: string) {
```

Comment:
I think eslint complains about missing return type on all exported functions

Reply:
Yeah, in the first commit I didn't have the configs quite right. ESLint now flags all of these, and I fixed them. Thanks.